src/retabled/core.cljc

```diff
@@ -77,16 +81,22 @@
   may be a function, keyword, etc, as specified by `(:valfn col-map)`"
   [col-map]
   (let [id (shared/idify (:headline col-map))
-        filter-address (:valfn col-map)]
+        filter-address (:valfn col-map)  #_[(:valfn col-map) :value]]
     [:input.filter {:id (str id "_filter")
-                    :value (or (@FILTER-MAP filter-address) "")
-                    :on-change #(swap! FILTER-MAP assoc filter-address (shared/get-value-from-change %))}]))
+                    :value (or (get-in @FILTER-MAP [filter-address :value]) "")
+                    :on-change #(swap! FILTER-MAP assoc filter-address {:value (shared/get-value-from-change %) :ignore-case? (:ignore-case? col-map true)})
+                    }])) ;Inline lambda
+
+(comment
+  {:name "Joe"
+   :name2 {:value "Joe"
+           :ignore-case? true}})
 
 (defn sort-click
   "Select sort field; if sort field unchanged, sort direction"
   [valfn SORT]
   (let [currently-selected (:selected @SORT)
-        swap-dir #(if (= <  %) > <)]
+        swap-dir #(if (= <  %) > <)] ;Inline lambda
```

Comment:
Please remove the `;Inline lambda` comments throughout the code here. That is a great way to mark up the code as you read it, but it  isn't necessary once the code goes live since experienced clojurians will be able to recognize the code without it saying that.